.appveyor.yml

```diff
@@ -1,21 +1,27 @@
+image: Visual Studio 2017
+
+clone_depth: 10
+
 environment:
-  nodejs_version: "8"
   MOCHA_TIMEOUT: "60000"
   NO_BREAKPOINTS: "1"
 
+cache:
+  - node_modules
```

Comment:
Should we specify [cache dependencies](https://www.appveyor.com/docs/build-cache/#cache-dependencies)?

Reply:
I'd say no. Even if `package.json` has evolved, it's faster to start from a slightly outdated cache than from scratch. And yarn knows how to manage this correctly.